rest_framework/fields.py

```diff
@@ -1397,7 +1397,8 @@ def __init__(self, choices, **kwargs):
     def to_internal_value(self, data):
         if data == '' and self.allow_blank:
             return ''
-
+        if isinstance(data, Enum) and str(data) != str(data.value):
```

Comment:
from the test failures it seems that Enum is not defined. you might forgot to import it? or could you check what is the issue?

Reply:
Thank you, I have added the import statement now.